components/Stars/index.scss

```diff
@@ -103,7 +103,8 @@
     }
 
     &:hover {
-      transform: none;
+      transition: transform .2s;
+      transform: scale(1.2);
```

Comment:
If you remove the transform none these styles will be applied automatically since they are already on the .star selector.